dart/tool/service.md

```diff
@@ -1113,8 +1117,7 @@ class Class extends Object {
   bool const;
 
   // The library which contains this class.
-  // TODO: This should be @Library, but the VM can return @Instance objects here.
-  @Object library;
+  @Library library;
```

Comment:
@bkonyi - I have seen the VM return InstanceRefs from here. I updated this version of the spec to allow ObjectRefs, but we should likely find where these are being returned from the VM and change that to LibraryRefs.